Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -2292,6 +2294,19 @@ RemoteMediaEngineConfigurationFactory& WebProcess::mediaEngineConfigurationFacto
 }
 #endif
 
+std::optional<IPC::Connection::UniqueID> WebProcess::networkProcessConnectionID()
```

Comment:
No real reason to use std::optional here. ObjectIdentifier already has an invalid state and you can boolean check it.